markdown/3.x/en/ff-asn.md

```diff
@@ -147,9 +147,11 @@ Slider groups are handled in a different way. To style them you need to use the
 <ff-asn-group-slider></ff-asn-group-slider>
 ```
 
-A slider can have a `[slot="groupCaption"]` attribute but instead of detailedLinks and hiddenLinks the slider group accepts an `ff-slider-control` and/or an `ff-slider` element.
+A slider can have a `[slot="groupCaption"]` attribute and a `[data-container="removeFilter"]` element but instead of `detailedLinks` and `hiddenLinks` the slider group accepts an `ff-slider-control`.
```

Comment:
```suggestion
A slider can have a `[slot="groupCaption"]` attribute, a `[data-container="removeFilter"]` element and a `ff-slider-control` element.
```

Reply:
I'm confused with `a` and `an` usage before `ff-*`. As far as I know we use `a` before words starting with consonants, `an` before words starting with vowels and whose pronounce like vowel-started (like "an hour"). Which is correct here: "a ff-component" or "an ff-component". The later one suits better for me.

Reply:
I am totally with you here 👍